wlauto/utils/android.py

```diff
@@ -100,6 +100,15 @@
     'UNINSTALL_SHORTCUT',
 ]
 
+GOOGLE_KEYBOARD_PACKAGE = 'com.android.inputmethod.latin'
+
+# List of known Google Keyboard versions that have problems
+# when key presses are used in AndroidUiAutoBenchmark workloads
+GOOGLE_KEYBOARD_BLACKLIST = [
```

Comment:
I would make this a dict with the key being the package and the values being lists of bad versions